src/execution/util/hash.cpp

```diff
@@ -0,0 +1,9 @@
+#include "execution/util/hash.h"
+
+#include "xxh3.h"  // NOLINT
```

Comment:
@amlatyrngom, can you expand on why you put NOLINT here?  Specifically, is it complaining about the include line or the file being included?  If it's the underlying header file, then this may be a more sustainable approach to maintaining our third party libs in general.

Reply:
Most likely a remnant of when this file was more complex. Remove it and see what happens.